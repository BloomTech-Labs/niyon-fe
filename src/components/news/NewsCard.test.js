import React from "react";
import { mount } from "enzyme";
import NewsCard from "./NewsCard";
import { MemoryRouter } from "react-router-dom";
import { UserContext } from "../../UserContext";
import { DarkModeContext } from "../../DarkModeContext";
import toJSON from "enzyme-to-json";
// import findByTestAttr from '../../tests/utils'

describe("<Search /> component testing", () => {
  const darkMode = false;
  const setDarkMode = jest.fn();
  const user = {};
  const setUser = jest.fn();
  let component;
  const profiles = [];
  beforeEach(() => {
    component = mount(
      <MemoryRouter initialEntries={["/news"]}>
        <UserContext.Provider value={{ user, setUser }}>
          <DarkModeContext.Provider value={{ darkMode, setDarkMode }}>
            <NewsCard />
          </DarkModeContext.Provider>
        </UserContext.Provider>
      </MemoryRouter>
    );
  });

  it("should pass snapshot testing", () => {
    expect(toJSON(component)).toMatchSnapshot();
  });

  
});


