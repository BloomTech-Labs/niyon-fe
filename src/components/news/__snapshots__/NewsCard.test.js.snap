// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewsCard /> component testing should pass snapshot testing 1`] = `undefined`;
